pycon/sponsorship/forms.py

```diff
@@ -46,8 +53,13 @@ class Meta:
         fields = [
             "name",
             "external_url",
+            "display_url",
+            "web_description",
+            "web_logo",
             "contact_name",
             "contact_emails",
+            "contact_phone",
+            "contact_address",
```

Comment:
I just noticed that the field order differs between the application form and the details/edit form. Would it be possible to combine these forms, or at least inherit one from the other?